### DISCUSSION TITLE
5.51.0 with mysql adapter with ruby-mysql 3.0.0 fails (Mysql.init missing)

### DISCUSSION THREAD

**Discussion by astounding** [jeremyevans/sequel#1801] [General]
I notice that ruby-mysql version 3.0.0 has done away with Mysql.init and Mysql#options methods that 5.51.0's mysql.rb adapter uses.  Is this something I should take up with the ruby-mysql folks, or whomever next has a chance to take a look at Sequel's MySQL adapter?  If I were better versed in adapter code I'd dive in myself, but I'm just a lowly Sequel user.  Until one gem or the other figures this out, I'll be stuck using 2.9.x versions of ruby-mysql.

Any insights appreciated!

Aaron out.

**Comment by flash-gordon** [comment]
See https://github.com/jeremyevans/sequel/issues/1795

**Reply by astounding** [reply]
Jeremy, thanks for making ruby-mysql work, including your newest tweaks for version 3.0.0's API!
